frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -100,18 +101,18 @@ const getStartUsedAndRemainingTime = (
 
     if (mission.endTime) {
         startTime = mission.startTime
-            ? format(new Date(mission.startTime), 'HH:mm')
-            : format(new Date(mission.endTime), 'HH:mm')
+            ? formatDateTime(new Date(mission.startTime), 'HH:mm')
+            : formatDateTime(new Date(mission.endTime), 'HH:mm')
         startDate = mission.startTime
-            ? format(new Date(mission.startTime), 'dd/MM/yyy')
-            : format(new Date(mission.endTime), 'dd/MM/yyy')
+            ? formatDateTime(new Date(mission.startTime), 'dd/MM/yyy')
+            : formatDateTime(new Date(mission.endTime), 'dd/MM/yyy')
         usedTimeInMinutes = mission.startTime
             ? differenceInMinutes(new Date(mission.endTime), new Date(mission.startTime))
             : 0
         remainingTime = 'N/A'
     } else if (mission.startTime) {
-        startTime = format(new Date(mission.startTime), 'HH:mm')
-        startDate = format(new Date(mission.startTime), 'dd/MM/yyy')
+        startTime = formatDateTime(new Date(mission.startTime), 'HH:mm')
+        startDate = formatDateTime(new Date(mission.startTime), 'dd/MM/yyy')
         usedTimeInMinutes = differenceInMinutes(Date.now(), new Date(mission.startTime))
```

Comment:
Need to use formattedDateTime/convertUTCDateToLocalDate in this comparison as well to avoid used time having an an offset of on hour

![image](https://github.com/equinor/flotilla/assets/33518988/37812d02-5617-4b08-a3ec-0d0566a29b40)